websocket/src/websocket.h

```diff
@@ -78,6 +78,30 @@ namespace dmWebsocket
         uint32_t m_Type:2;
     };
 
+    struct HttpHeader
+    {
+        const char* m_Key;
+        const char* m_Value;
+
+        HttpHeader() = delete;
```

Comment:
Although some compilers have their default set to C++11, we aren't (yet) officially supporting more than C++98. If the extension compiles stand-alone, for all platforms, it's good.

Reply:
removed c++11 constructors